src/models/db.js

```diff
@@ -36,6 +36,8 @@ require('./Game');
 require('./MainSettings');
 require('./GeneralCounters');
 
+require('./Player');
+
```

Comment:
remove blank line (maybe install a formatter for JS and HTM, which handles such things automatically) 
i recommend: Prettier - Code formatter